web/template/layout/refund.gohtml

```diff
@@ -20,8 +20,14 @@
                     <tr class="govuk-table__row">
                         <th scope="row" class="govuk-table__header">Method of refund:</th>
                         <td class="govuk-table__cell">
-                            {{ range $.PaymentSources }}
-                                {{ if eq .Handle $p.Source }}{{ .Label }}{{ end }}
+                            {{ if eq "MAKE" $p.Source }}
+                                GOV.UK Pay (online - Make an LPA)
+                            {{ else if eq "ONLINE" $p.Source }}
+                                GOV.UK Pay (card payment)
+                            {{ else }}
```

Comment:
I think it would be better to do this in the PaymentSource RefData in the API, then we have a single source of language and don't need to do translations like this in every frontend. But that might be easier as a follow-up ticket since the current translations are all "Paid X" so we either need to add a second translation property (like `isSelectable`) or rework the translations to be more reusable

Reply:
Good shout, i've created a ticket here: https://opgtransform.atlassian.net/jira/software/projects/VEGA/boards/170/backlog?selectedIssue=VEGA-1712